helm/ibft2-with-permissioning/besu/templates/besu-node-permissions-configmap.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ template "besu.fullname" . }}-node-permissions-configmap
+  labels:
+    app: {{ template "besu.name" . }}
+    chart: {{ template "besu.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+    namespace: {{ .Values.namespace }}
+  namespace: {{ .Values.namespace }}
+data:
+  nodes-whitelist.yml: |-
+    nodes-whitelist=[
+      "enode://{{ trimPrefix "0x" .Values.validators.validator1.publicKey }}@{{ template "besu.name" . }}-validator1-0.{{ template "besu.name" . }}-validator1.{{ .Values.namespace }}.svc.cluster.local:30303",
+      "enode://{{ trimPrefix "0x" .Values.validators.validator2.publicKey }}@{{ template "besu.name" . }}-validator2-0.{{ template "besu.name" . }}-validator2.{{ .Values.namespace }}.svc.cluster.local:30303",
+      "enode://{{ trimPrefix "0x" .Values.validators.validator3.publicKey }}@{{ template "besu.name" . }}-validator3-0.{{ template "besu.name" . }}-validator3.{{ .Values.namespace }}.svc.cluster.local:30303",
+      "enode://{{ trimPrefix "0x" .Values.validators.validator4.publicKey }}@{{ template "besu.name" . }}-validator4-0.{{ template "besu.name" . }}-validator4.{{ .Values.namespace }}.svc.cluster.local:30303",
+      "enode://{{ trimPrefix "0x" .Values.nodes.node1.publicKey }}@{{ template "besu.name" . }}-node1-0.{{ template "besu.name" . }}-node1.{{ .Values.namespace }}.svc.cluster.local:30303"
```

Comment:
just to be sure that this line which is different from the others must be there

Reply:
Yep, all the docs and readme have 4 validators & 1 node so I added a key for 1 node and put it in here as well to stay consistent.